hpat/tests/test_series.py

```diff
@@ -611,6 +611,42 @@ def test_impl(S):
         S = pd.Series([1.0, 2., 3., 4., 5.])
         pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
 
+    def test_series_combine(self):
```

Comment:
Tests for data types other than `float64` would be useful. Also, input series with different types would be good. For example, S1 with integers and S2 with floats can make sure output type is not just looking at S1's type.